Issue 7426 (#7438)
umbraco/UmbracoDocs: 16/umbraco-cms/reference/content-delivery-api/additional-preview-environments-support.md

# Additional preview environments support

{% hint style="warning" %}

> The original contents of this article, explaining how to change the backoffice UI, does not apply anymore to Umbraco 15 and above and has been removed.

Reviewer: ```suggestion
The original content of this article, explaining how to change the backoffice UI, does not apply anymore to Umbraco 15 and above and has been removed.
```

> The way to achieve this beyond Umbraco 14 is to create and register an extension to modify the backoffice UI preview button.

Reviewer 2: Can we link to anywhere to show how this is done?  Even if we don't have a specific article to show how to modify the backoffice UI preview button, at least to a page generally talking about client-side extensions.

Author: Thanks @AndyButland  for the review I have added the link 😊

{% endhint %}

With Umbraco, you can save and preview draft content before going live. The preview feature allows you to visualize how a page will look like once it is published, directly from within the backoffice. This is also possible for the Content Delivery API data. You can extend the preview functionality in the backoffice by configuring external preview URLs for client libraries consuming the Content Delivery API.

{% hint style="info" %}

> To get familiar with the preview functionality in the Content Delivery API, please refer to the [Preview concept](https://docs.umbraco.com/umbraco-cms/reference/content-delivery-api#preview) section.

Reviewer: ```suggestion
To get introduced to the preview functionality in the Content Delivery API, please refer to the [Preview concept](https://docs.umbraco.com/umbraco-cms/reference/content-delivery-api#preview) section.
```

{% endhint %}

{% hint style="info" %}

> The support for configuring additional preview environments in the Content Delivery API was introduced in version 12.3.

Reviewer: ```suggestion
Support for configuring additional preview environments in the Content Delivery API was introduced in version 12.3.
```

{% endhint %}

{% hint style="info" %}

> There are plans to simplify this process. In an upcoming major version of Umbraco, a UI will be introduced, allowing you to configure these custom preview URLs directly from the backoffice.

Reviewer: ```suggestion
An upcoming release of Umbraco will allow users to configure custom preview URLs directly in the backoffice.
```
